com.github.Nheko.json

```diff
@@ -0,0 +1,210 @@
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "./bootstrap.sh --with-libraries=random,thread,system,iostreams,atomic,chrono,date_time,regex --prefix=/usr",
+        "./b2 -d0 variant=release link=shared threading=multi --layout=system",
+        "./b2 -d0 install"
+      ],
+      "buildsystem": "simple",
+      "name": "boost",
+      "sources": [
+        {
+          "sha256": "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9",
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DBUILD_LIB_TESTS=OFF -DBUILD_LIB_EXAMPLES=OFF -DCMAKE_BUILD_TYPE=Release",
+        "cmake --build build --target install --config Release"
+      ],
+      "buildsystem": "simple",
+      "name": "mtxclient",
+      "sources": [
+        {
+          "sha256": "184d155ec731a23750583c8a69a8cadda8af70994fea66c20f8231fb3072ddad",
+          "type": "archive",
+          "url": "https://github.com/mujx/mtxclient/archive/ca66424489b4d3e6ea447b853157fa88f0b849b3.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DTWEENY_BUILD_DOCUMENTATION=OFF -DTWEENY_BUILD_EXAMPLES=OFF",
+        "cmake --build build --config Release --target install"
+      ],
+      "buildsystem": "simple",
+      "name": "tweeny",
+      "sources": [
+        {
+          "sha256": "482857256a7235646004682912badb6521d361ed6987c8ebdae7986bf64ce694",
+          "type": "archive",
+          "url": "https://github.com/mobius3/tweeny/archive/43f4130f7e4a67c19d870b60864bc2862c19b81f.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DLMDBXX_INCLUDE_DIR=.deps/lmdbxx",
+        "cmake --build build --config Release"
+      ],
+      "buildsystem": "simple",
+      "name": "nheko",
+      "post-install": [
+        "./install-resources.sh"
+      ],
+      "sources": [
+        {
+          "sha256": "bfc9d4e3715241c1878dbf16ff8cd5df18651e4dd32f780e0957d3724de57592",
+          "type": "archive",
+          "url": "https://github.com/mujx/nheko/archive/v0.5.2.tar.gz"
+        },
+        {
+          "path": "install-resources.sh",
+          "type": "file"
+        },
+        {
+          "path": "nheko.appdata.xml",
+          "type": "file"
+        },
+        {
+          "dest": ".deps/lmdbxx",
+          "sha256": "93721132bbf5045d38ad62de2997655e9984c48ea5c9886746d42128f4b26fbd",
+          "type": "archive",
+          "url": "https://github.com/bendiken/lmdbxx/archive/0b43ca87d8cfabba392dfe884eb1edb83874de02.tar.gz"
+        }
+      ]
+    }
+  ],
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.10",
+  "sdk": "org.kde.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.gcc8"
+  ],
+  "writable-sdk": true
```

Comment:
Remove this.

Reply:
This is needed because third-party libraries create `/usr/local` as the installation dir.